elements/runes.lua

```diff
@@ -80,27 +80,46 @@ local function descSort(runeAID, runeBID)
 	end
 end
 
-local function UpdateColor(element, runeID)
+local function UpdateColor(self, event)
+	local element = self.Runes
+
 	local spec = GetSpecialization() or 0
 
 	local color
-	if(spec > 0 and spec < 4 and element.colorSpec) then
-		color = element.__owner.colors.runes[spec]
+	if(spec  > 0 and spec < 4 and element.colorSpec) then
```

Comment:
One extra space here :)
Also, tbh, I don't think we need to keep the `.colorSpec` option as Blizzard's defaults are now coloring by spec and there are no more different rune types for the last two expansions.

Reply:
Yeah, I saw it too, just didn't want to have a separate commit for it :D